charts/tenant-base/chart/values.yaml

```
@@ -1,73 +1,109 @@
-image:
-  repository: ghcr.io/stefanprodan/podinfo
-  pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: 6.2.1
+# Allows for renaming of the chart, this is advised since the pods would include 'tenant-base' as the name otherwise.
+fullnameOverride: foobar
 
-imagePullSecrets: []
+# A list of deployments, this allows for multiple deployments in the same chart.
+deployments:
+  # Name of the deployments - this is appended on the deployment and service name.
+  - name: podinfo
+    image:
+      # The repository containing the image
+      repository: ghcr.io/stefanprodan/podinfo
+      # The pullPolicy
+      # * IfNotPresent - If the image is not cached on the device
+      # * Always         - Will always try to fetch the newest image
+      # * Never          - Will not try to fetch an image - if image is not present on the machine startup fails
```

The indentation is off:
```suggestion
      # * IfNotPresent - If the image is not cached on the device
      # * Always       - Will always try to fetch the newest image
      # * Never        - Will not try to fetch an image - if image is not present on the machine startup fails
```